README.md

```diff
@@ -340,6 +340,54 @@ LIT_SERVER_API_KEY=supersecretkey python main.py
 
 Clients are expected to auth with the same API key set in the `X-API-Key` HTTP header.
 
+</details>
+
+<details>
+  <summary>Dynamic batching</summary>
+&nbsp;
+
+FastServe can combine individual requests into a batch to enhance throughput.
+To enable batching, you need to set the `max_batch_size` argument and implement the `LitAPI.batch` and `LitAPI.unbatch`
```

Comment:
`batch` and `unbatch` has the default implementation only in case of the input is either torch.Tensor or np.ndarray. There might be cases where the input might be string or some primary data type.

Reply:
so in this particular case, to remove `batch` and `unbatch`, `decode_request` will have to return a numpy or torch tensor.